spec/models/document_spec.rb

```diff
@@ -131,4 +155,14 @@
       end
     end
   end
+
+  describe "#convert_heic_upload_to_jpg!" do
+    it "converts a heic attachment to jpg" do
+      document = create :document, upload_path: Rails.root.join("spec", "fixtures", "attachments", "IMG_4851.HEIC")
+
+      expect {
+        document.convert_heic_upload_to_jpg!
+      }.to change { document.upload.attachment.filename.extension }.from("HEIC").to("jpg")
```

Comment:
This test could be more exhaustive - perhaps we can ask the `mimemagic` library that we have in the project what the file's detected MIME type is. Not a blocker.